### ISSUE TITLE
Matriz de inimigos

### ISSUE THREAD

**Issue by olivatooo** [UncreativeGames/captain-cleiton#15]
A classe `TileMap` deve gerar uma matriz de inimigos.

**Comment by vitoor-s** [comment]
isso permitirá apenas um inimigo por quadrante 32x32 certo? a movimentação deles é feita pelos pixels, é esse o desejado?

**Comment by olivatooo** [comment]
@vitoor-s Exatamente! Mas lembrando que com #34 podemos alterar essa "constante" de 32x32
